src/PrizePool.sol

```diff
@@ -435,73 +506,118 @@ contract PrizePool {
+    */
     function _getVaultUserBalanceAndTotalSupplyTwab(address _vault, address _user, uint256 _drawDuration) internal view returns (uint256 twab, uint256 twabTotalSupply) {
-        {
-            uint32 endTimestamp = uint32(lastCompletedDrawStartedAt_ + drawPeriodSeconds);
-            uint32 startTimestamp = uint32(endTimestamp - _drawDuration * drawPeriodSeconds);
-
-            twab = twabController.getAverageBalanceBetween(
-                _vault,
-                _user,
-                startTimestamp,
-                endTimestamp
-            );
-
-            twabTotalSupply = twabController.getAverageTotalSupplyBetween(
-                _vault,
-                startTimestamp,
-                endTimestamp
-            );
-        }
+        uint32 endTimestamp = uint32(lastCompletedDrawStartedAt_ + drawPeriodSeconds);
+        uint32 startTimestamp = uint32(endTimestamp - _drawDuration * drawPeriodSeconds);
+
+        twab = twabController.getAverageBalanceBetween(
+            _vault,
+            _user,
+            startTimestamp,
+            endTimestamp
+        );
+
+        twabTotalSupply = twabController.getAverageTotalSupplyBetween(
+            _vault,
+            startTimestamp,
+            endTimestamp
+        );
     }
 
+    /**
+    * @notice Returns the time-weighted average balance (TWAB) and the TWAB total supply for the specified user in the given vault over a specified period.
+    * @param _vault The address of the vault for which to get the TWAB.
+    * @param _user The address of the user for which to get the TWAB.
+    * @param _drawDuration The duration of the period over which to calculate the TWAB, in number of draw periods.
+    * @return The TWAB and the TWAB total supply for the specified user in the given vault over the specified period.
+    */
     function getVaultUserBalanceAndTotalSupplyTwab(address _vault, address _user, uint256 _drawDuration) external view returns (uint256, uint256) {
         return _getVaultUserBalanceAndTotalSupplyTwab(_vault, _user, _drawDuration);
     }
 
-    function _getVaultPortion(address _vault, uint32 drawId_, uint32 _durationInDraws, SD59x18 _alpha) internal view returns (SD59x18) {
+    /**
+    * @notice Calculates the portion of the vault's contribution to the prize pool over a specified duration in draws.
+    * @param _vault The address of the vault for which to calculate the portion.
+    * @param drawId_ The draw ID for which to calculate the portion.
+    * @param _durationInDraws The duration of the period over which to calculate the portion, in number of draws.
+    * @param _smoothing The alpha value to use for calculating the portion.
+    * @return The portion of the vault's contribution to the prize pool over the specified duration in draws.
+    */
+    function _getVaultPortion(address _vault, uint32 drawId_, uint32 _durationInDraws, SD59x18 _smoothing) internal view returns (SD59x18) {
```

Comment:
Should be `_drawId` instead of `drawId_` no?
```suggestion
    function _getVaultPortion(address _vault, uint32 _drawId, uint32 _durationInDraws, SD59x18 _smoothing) internal view returns (SD59x18) {
```